### PR TITLE
[Doc] Update README for local builds using make

### PR DESCRIPTION
**Pull request by black-dragon74**
Fixes: #78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,31 +46,17 @@ run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gath
...
 You need to be autheticated against the OpenShift registry CI, registry.ci.openshift.org in order to be able to pull the images in the `Dockerfile`.
 
-Before proceeding make sure you have `oc` (OpenShift cli) and `jq` (optional) installed.
+Before proceeding make sure you have `oc` (OpenShift cli) installed.
 
 Follow these steps to authenticate:
 
 - Navigate to the OpenShift console for `app.ci` cluster located [here](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/).
 - Login using SSO
 - Go to your profile name in the top right and select "Copy login command".
```
Comment (yati1998) [inline README.md:55]: instead of copy the login command it would be better to mention copy the token from the login command.

Comment (black-dragon74) [reply README.md:55]: Ah! The menu is actually labeled "copy login command" hence the statement.

Regards

Comment (black-dragon74) [reply README.md:55]: I think there is a slight misunderstanding, the drop down menu is called "copy login command". 

Hence it is written like that. The next step tells user to copy the displayed token. 

TLDR; You cannot get to token screen without clicking on "copy login command" from the overflow menu. 

Hope this helps clarify.

Comment (yati1998) [reply README.md:55]: ohh ops, didn't notice that. cool LGTM then.
